### PR TITLE
[FIX] Update environment variable name and link to query tool README

### LINKED ISSUE #179: Query tool standalone version code snippet did not work
### Description

As in the end it worked out, it might be a documentation issue more than an issue of the query tool itself.

The command
`docker run -d -p 3000:3000 --env API_QUERY_URL=http://localhost:8000/ --name query_tool neurobagel/query_tool:latest`

did not work out – I couln’t start the query tool container (I always used the default port mappings). 

NiDkbZ

After changing the `API_QUERY_URL` to `NB_API_QUERY_URL` I could start it and inspect the `app/vite.config.ts` where I figured out that the returned port is `5173`.

After some trial and error the command:
`docker run -d -p 3000:5173 --env NB_API_QUERY_URL=http://localhost:8000/ --name query_tool neurobagel/query_tool:latest`  

got me to what I would see as desired result:
image

Is this then the right command or did I do something wrong and it worked out accidentally?

### Relevant documentation pages

https://neurobagel.org/infrastructure/#deploy-a-graphical-query-tool

### Additional details

_No response_

### PR DESCRIPTION
**Pull request by alyssadai**
- Closes #179 

Changes proposed in this pull request:

- `API_QUERY_URL` replaced by `NB_API_QUERY_URL`
- Using [mkdocs-include-markdown-plugin](https://github.com/mondeja/mkdocs-include-markdown-plugin), query tool documentation page contents replaced by link to query tool [README](https://github.com/neurobagel/query-tool/blob/main/README.md) sections where appropriate for easier syncing

## Checklist
_Please leave checkboxes empty for PR reviewers_

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`) _see our [Contributing Guidelines](https://neurobagel.org/contributing/CONTRIBUTING/#pull-request-guidelines) for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [x] Checks pass

### CODE REVIEW

File: docs/infrastructure.md [hunk 1]
```diff
@@ -570,10 +570,10 @@ as a standalone docker container.
 
 
 ```bash
-docker run -d -p 3000:3000 --env API_QUERY_URL=http://localhost:8000/ --name query_tool neurobagel/query_tool:latest
+docker run -d -p 3000:3000 --env NB_API_QUERY_URL=http://localhost:8000/ --name query_tool neurobagel/query_tool:latest
```
Comment (rmanaem) [inline docs/infrastructure.md:573]: Forgot to mention this as well, Vite by default uses port `5173` so:
```suggestion
docker run -d -p 3000:5173 --env NB_API_QUERY_URL=http://localhost:8000/ --name query_tool neurobagel/query_tool:latest
```

File: docs/infrastructure.md [hunk 2]
```diff
@@ -570,10 +570,10 @@ as a standalone docker container.
...
 ```
 
-Make sure to replace the value of `API_QUERY_URL` with the `IP:PORT` or domain name of the
+Make sure to replace the value of `NB_API_QUERY_URL` with the `IP:PORT` or domain name of the
 new neurobagel node-API you just deployed!
 
 If using the default port mappings for the query tool (`-p 3000:3000` in above command),
```
Comment (rmanaem) [inline docs/infrastructure.md:579]:
```suggestion
If using the default port mappings for the query tool (`-p 3000:5173` in above command),
```

File: docs/federate.md [hunk 3]
```diff
@@ -172,7 +172,7 @@ services:
     ports:
       - "${NB_QUERY_PORT_HOST:-3000}:3000"
```
Comment (rmanaem) [inline docs/federate.md:173]:
```suggestion
      - "${NB_QUERY_PORT_HOST:-3000}:5173"
```

**Review by rmanaem — COMMENTED** [review]
Thanks for taking this on @alyssadai!
I think the changes look good and the `include-markdown` seems to be working as expected. There is only one thing and that's the default port query tool runs on which since we made the transition to React+Vite has changed to `5173`. However I think it's better to change this in the query tool to `3000` so we don't have to update it anywhere else. wdyt?

**Comment by alyssadai** [comment]
Hey @rmanaem, thanks for catching that!

I think since the deployment recipes and docs have been already modified anyways related to this variable, I would say we can go ahead and change the port to the one used by the container to the Vite default for now. This way, at least our current instructions will work in the interim before we switch to the single-Compose recipe, and we can decide when we refine that new single-Compose file whether we want to keep this port or not.

Will request your review on a PR for this port change in the `recipes` repo !

**Review by rmanaem — APPROVED** [review]
🧑‍🍳

**Pull request merged**